satpy/etc/readers/aws_l1c_nc.yaml

```diff
@@ -0,0 +1,375 @@
+reader:
```

Comment:
Filename should be `mws_l1c_nc.yaml` and the metadata should also be updated accorindingly.